### PR TITLE
Fix CHTC repo non-HTCondor package installs (HTCONDOR-84)

### PR DESCRIPTION
**Pull request by brianhlin**
https://opensciencegrid.atlassian.net/browse/HTCONDOR-84

Bonus removal of unnecessary `grid-certificates` OSG package requirement

### CODE REVIEW

**Comment by matyasselmeci** [comment]
We still have GSI as one of the default auth methods, right? Is auth going to work without certificates, or will auth bail out due to GSI being unhappy? Non-OSG people aren't going to install osg-ce...

**Comment by brianhlin** [comment]
Correct, we tell non-OSG users to install certificates separately since there isn't an equivalent `osg-certificates` package elsewhere since that's just not how they manage site CAs outside of the OSG: https://htcondor-ce.readthedocs.io/en/latest/installation/htcondor-ce/#before-starting

Plus, the requirement was only for OSG packaging so this has no effect for the non-OSG case.

**Comment by matyasselmeci** [comment]
Looks like you'll need to install "python3-rpm" to get the osg tests to work. Dunno about the uw tests missing cagen.

File: rpm/htcondor-ce.spec [hunk 1]
```diff
@@ -141,12 +126,8 @@ Requires: blahp
 Group: Applications/System
 Summary: Default routes for submission to Slurm
 Requires: %{name} = %{version}-%{release}
-
-%if 0%{?uw_build}
 Requires: /usr/libexec/condor/glite/bin/batch_gahp
```
Comment (timtheisen) [inline rpm/htcondor-ce.spec:129]: I think you missed this one.

Comment (brianhlin) [reply rpm/htcondor-ce.spec]: Good catch!

**Comment by brianhlin** [comment]
@matyasselmeci since the CI is hung up for whatever reason and this is a straightforward, urgent fix, can you re-review? I'd rather not solve all the CI issues in this PR.

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**